scripts/merge_libraries_test.py

```diff
@@ -0,0 +1,277 @@
+# Copyright 2018 Google LLC
```

Comment:
This should be 2022 if this file is new (not from g3)

Reply:
It's from g3